sormas-api/src/main/java/de/symeda/sormas/api/event/TypeOfPlace.java

```diff
@@ -17,47 +17,39 @@
  *******************************************************************************/
 package de.symeda.sormas.api.event;
 
-import java.util.ArrayList;
+import java.util.Arrays;
 import java.util.List;
 
 import de.symeda.sormas.api.i18n.I18nProperties;
+import de.symeda.sormas.api.utils.HideForCountriesExcept;
 
 public enum TypeOfPlace {
 
-	FACILITY(true),
-	FESTIVITIES(false),
-	HOME(true),
-	MEANS_OF_TRANSPORT(false),
-	PUBLIC_PLACE(false),
-	SCATTERED(false),
-	UNKNOWN(false),
-	OTHER(false);
-
-	private static List<TypeOfPlace> typesUsableForCases = null;
-
-	private boolean usableForCases;
-
-	TypeOfPlace(boolean usableForCases) {
-		this.usableForCases = usableForCases;
-	}
-
-	public boolean isUsableForCases() {
-		return usableForCases;
-	}
-
-	public static List<TypeOfPlace> getTypesOfPlaceForCases() {
-		if (typesUsableForCases == null) {
-			typesUsableForCases = new ArrayList<>();
-			for (TypeOfPlace typeOfPlace : values()) {
-				if (typeOfPlace.isUsableForCases()) {
-					typesUsableForCases.add(typeOfPlace);
-				}
-			}
-		}
-		return typesUsableForCases;
-	}
+	FACILITY,
+	@HideForCountriesExcept(countries = {})
```

Comment:
Parameters can be removed from these annotations

Reply:
The parameter has to be the empty list her, since these values are not intended to be shown at all except for `ActicityAsCase` for instances in Germany.
Without the empty list, the parameter defaults to "de", so these values would be shown by default wherever `TypeOfPlace` is used on instances in Germany.

Reply:
Ah, I see! Makes sense, thanks!